### ISSUE TITLE
Allow Automation policy applied items to be disabled per-agent

### ISSUE THREAD

**Issue by silversword411** [amidaware/tacticalrmm#1196]
First I think there's 2 switches on an automation policy.

Enabled and enforced. If enforced is on (does it do anything right now?), you can't turn any check or task off (greyed out and unchangeable like it is now). I think the intention of forced is for the policy to work like AD GPO enforced?

Second, add a power button (enabled) checkbox for all tasks and scripts (will also be useful for quick/easy enabling/disabling items). 

Third, when applying a policy it will default check and apply all the policy scripts and tasks, but you would then be able to turn off (disable) any check or task on a per-agent check/task basis if needed.

**Comment by sadnub** [comment]
There isn't a super easy way to do this because there are no check/task objects on the agent itself. They are dynamically calculated and cached. Enforced was meant to be able to override conflicting checks, but I might be able to allow per check/task modifications. I'll have to dig more

**Comment by dinger1986** [comment]
I don't understand why you would want to do that, you can apply a different policy to a machine so if you have a blank policy just assign that or whatever other policy you want?

I certainly wouldn't want any of my techs to be able to disable a check by accident or if alerts are annoying them.

**Comment by sadnub** [comment]
Yeah you could exclude the agent from the other policy then apply an agent level policy

**Comment by silversword411** [comment]
> there are no check/task objects on the agent itself

The GUI is deceptive then, looked like there were already controls that were force-enabled to "apply" the check/task to an agent...and it was forced because it came from: policy (shield).

I was hoping it was just a flag that could then be toggled via the gui per check/task...and it was an easy fix :)

**Comment by silversword411** [comment]
@sadnub one day if you'd like to discuss I think I've thought thru all the policy logic and have a way you could have policy, when to apply and calculate policy, how to track and override per agent etc.

**Comment by dinger1986** [comment]
I was thinking about it as well, so almost have a policy as a template which can be overwritten per agent (provided the user has permission) and if its enforced it cant be overwritten, something like that anyway
